torch/_inductor/codegen/triton_split_scan.py

```diff
@@ -102,7 +105,18 @@ def scan(self, dtypes, combine_fn, values):
 
         assert len(self.numels) == 2, "Unexpected tiling"
         min_rblock = config.triton.min_split_scan_rblock
-        max_blocks = prod(self.numels[:-1]) * CeilDiv(self.numels[-1], min_rblock)
+        is_reduction_dim = [var[0] == "r" for var in self.numels]
+        reduction_numel = math.prod(
+            numel
+            for is_reduction, (var, numel) in zip(is_reduction_dim, self.numels.items())
```

Comment:
nit: either remove `is_reduction_dim`, or use `self.numels.values()`. Having both vars and `is_reduction_dim` is redundant.